confdb/aclocal_cc.m4

```diff
@@ -845,7 +849,7 @@ int main( int argc, char *argv[] )
     struct { float b; char a; } float_char;
     struct { char a; double b; } char_double;
     struct { double b; char a; } double_char;
-#ifdef HAVE_LONG_DOUBLE
+#if SIZEOF_LONG_DOUBLE > 0
```

Comment:
`long double` seems to have been there since C89.  Again, I recommend we abort if it's not present.

https://en.wikipedia.org/wiki/Long_double

Reply:
Ok.